lib/jsdom/living/nodes/HTMLImageElement-impl.js

```diff
@@ -29,10 +64,10 @@ class HTMLImageElementImpl extends HTMLElementImpl {
   }
 
   get height() {
-    if (this.hasAttribute("height")) {
-      return conversions["unsigned long"](this.getAttribute("height"));
-    }
-    return 0;
+    // Just like on browsers, if no width / height is defined, we fall back on the
+    // dimensions of the internal image data.
+    return this.hasAttribute("height") ?
```

Comment:
Yeah, Chrome has wrong IDL for HTMLCanvasElement; see the FIXME comment in https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/Source/core/html/HTMLCanvasElement.idl&q=HTMLCanvasElement.idl&sq=package:chromium&l=1